jujugui/static/gui/src/app/components/generic-button/_generic-button.scss

```diff
@@ -1,102 +1,198 @@
-.generic-button {
-    @extend %button;
-    padding-left: 10px;
-    padding-right: 10px;
+////
+/// @author       Web Team at Canonical Ltd
+/// @link         http://ubuntudesign.github.io/vanilla-framework/docs/#mixin-gui-buttons
+/// @since        0.0.3
+////
+
+/// Default blockquote styling
+/// @group Buttons
+@mixin gui-buttons {
+  @include gui-neutral-button;
+  @include gui-positive-button;
+  @include gui-destructive-button;
+  @include gui-base-button;
+  @include gui-deployment-button;
+}
 
-    &--type-confirm {
-        @extend %button-orange;
-    }
+/// Neutral button styling
+/// @group Buttons
+/// @example
+/// <a class="button--neutral">
+///   ...
+/// </a>
+@mixin gui-neutral-button {
+  .button--neutral,
+  .button--inline-neutral {
+    @extend %button-pattern;
+    color: $cool-grey;
+    background-color: transparent;
+    border: 1px solid $mid-grey;
 
-    &--type-blue {
-        @extend %button-blue;
+    &:hover {
```

Comment:
We should probably have active and focus states for these buttons as well, might be worth checking with design about what those states should look like.